### ISSUE TITLE
Hi dude

### ISSUE THREAD

**Issue by Alvazz** [HEEV/motor-controller-2018#1]
What is the final operation effect of this project? I was surprised to see that you mentioned staying away from TMC4671?

**Comment by sellicott** [comment]
No, we used that chip in the design. It worked pretty well, it's just kind of a pain to solder. Also, at least when we used it in our project, there were a few undocumented hardware bugs. I am guessing at least a few of them have been fixed at this point though.
